vignettes/wrapping-apis.Rmd

```diff
@@ -397,11 +417,13 @@ req <- request("https://api.nytimes.com/svc/books/v3") %>%
 
 Putting together all the pieces above yields a function something like this:
 
+
 ```{r}
 nytimes_books <- function(api_key, path, ...) {
   request("https://api.nytimes.com/svc/books/v3") %>% 
     req_url_path_append("/reviews.json") %>% 
     req_url_query(..., `api-key` = api_key) %>% 
+    # JENNY COMMENT HERE
```

Comment:
I see you do model this in a later example, but I wonder if you want to predefine the error-body-handling function? Either outside of `nytimes_books()` or inside. Because I can't imagine these would ever stay simple enough to define inline like this. Also this feels pretty hostile to writing tests.

Reply:
Good call. I do that later on, but not here.